frontend/views/partials/searchResult.html

```diff
@@ -0,0 +1,67 @@
+<div class="col-xs-12 col-sm-11">
```

Comment:
@Veegas 
Please add some html documentation to indicate where a section starts, ends and what it does
like this
<!--container of books starts here-->
html code
<1--container of books ends here-->